.github/workflows/major-version-check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/github-script@v7
         with:
           script: |
-            const prNumber = context.payload.pull_request.number;
+            const prNumber = context.payload?.pull_request?.number || context.payload?.issue?.number;
```

Comment:
❓ is the `context.payload?.issue?` right here?

Reply:
If it's triggered by the `issue_comment` trigger, then the `pull_request` property on payload is `undefined` 😞